storage/perfschema/pfs_events_waits.h

```diff
@@ -54,6 +54,23 @@ enum events_waits_class
 /** A wait event record. */
 struct PFS_events_waits : public PFS_events
 {
+  /** Flags */
+  ulong m_flags;
```

Comment:
This one is 32bits on Windows, so it should be after `size_t`.

Reply:
`size_t` too: https://godbolt.org/z/yFec-m It's like `sizeof(void*)`, IIRC.

Reply:
So, on Linux x86 `ulong` should go before `size_t`.

Reply:
Why? `ulong` should be 32bits on ILP32?

Reply:
You're right. Sorry for my inattention.